cdap-app-fabric/src/test/java/io/cdap/cdap/internal/app/services/http/handlers/PreferencesHttpHandlerInternalTest.java

```diff
@@ -47,9 +50,12 @@ public static void init() {
     store = getInjector().getInstance(Store.class);
   }
 
-  private void addApplication(String namespace, Application app) {
+  private void addApplication(String namespace, Application app) throws ConflictException {
     ApplicationSpecification appSpec = Specifications.from(app);
-    store.addApplication(new ApplicationId(namespace, appSpec.getName()), appSpec);
+    ApplicationMeta meta = new ApplicationMeta(appSpec.getName(), appSpec, new ChangeDetail(null, null, null,
```

Comment:
Move the change detail to a new line so the constructor fields are in the same line